roles/slapd/files/saslauthd/run

```diff
@@ -0,0 +1,8 @@
+#!/bin/sh
+[ -r ./conf ] && . ./conf
+: ${SASLAUTHDMECH:=Kerberos5}
```

Comment:
While it is not possible for this to have a valid value and have a space in it, please quote it nonetheless.  As written this would be good to merge back to upstream, perhaps the one we version should be simpler until an upstream merge can be completed?